src/routes/index.js

```diff
@@ -89,6 +89,52 @@ const getDevice = (device) => {
   });
 }
 
+async function queryGitHubReleasesApi(requestedInstallerPackage) {
+  return new Promise ((resolve, reject) => {
+    request.get({
+      url: "https://api.github.com/repos/ubports/ubports-installer/releases/latest",
+      json: true,
+      headers: { 'User-Agent': 'request' }
+    },
+    (err, res, bod) => {
+      if (!err && res.statusCode === 200) {
+        for (asset in bod.assets) {
+          if(bod.assets[asset].browser_download_url.split('.').pop().toLowerCase() == requestedInstallerPackage)
+            resolve(bod.assets[asset].browser_download_url)
+        }
+        reject();
+      }
+    });
+  });
+}
+
+async function getInstaller(installerPackage) {
+  return new Promise ((resolve, reject) => {
+    try {
+      switch (installerPackage) {
+        case "linux":
+        case "snap":
+          resolve("https://snapcraft.io/ubports-installer");
```

Comment:
Do we not want to show the options to use appimage and deb?

Reply:
just asking since it take the argv "linux"

Reply:
I don't want to "show" anything. The idea is to just have simple quick redirects that will work for everyone. so linux redirects to a linux package that works everywhere. That is snap. If someone wants appimage, deb or any of the other packages available on github, they'll get them automatically.

Reply:
They don't need a switch case, since they are always downloadable files from the github release